build.gradle

```diff
@@ -91,12 +91,12 @@ subprojects {
         jvmArgs += ["-Xmx1024m", "-XX:+StartAttachListener", "-XX:+HeapDumpOnOutOfMemoryError", "-XX:HeapDumpPath=/var/log/uaa-tests.hprof"]
 
         retry {
-            def retryFailOnPassedAfterRetry = Boolean.parseBoolean(
+            def failOnPassedAfterRetryProperty = Boolean.parseBoolean(
```

Comment:
I really appreciate the effort of trying to make this variable easier to understand.

I don't understand the name of this new variable though.

If you think this one is better than the old one, it's better this way. I think that we should be able to do even better though.

Reply:
I think the old one was very misleading - implying that it would affect whether a retry would happen, which is not what it does. It only affects whether the test will be reported as a pass or fail based on whether it failed during any of the retries.

I'm sure we can improve on the new name. I did a bit of experimenting because it's not clear why this variable is needed as an intermediary at all. Assigning directly to the failOnPassedAfterRetry property and using it in a boolean did not work, so I kept the variable and just tried to give it a better name.

Reply:
Renamed to failOnPassedAfterRetrySystemProperty and added a comment.